### DISCUSSION TITLE
Disabled input should have darker background color

### DISCUSSION THREAD

**Discussion by finwe** [primefaces/community#593] [PrimeNG] [answered]
Disabled input should have darker background color. Currently, the change in border color is so subtle that when empty, it can hardly be distinguished from enabled input unless hovered over (which is too late).

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's made by design. You can customize the theme by using primeng-sass-theme or targeting the disabled class of the component.

**Reply by finwe** [reply]
That I can, but I still believe the default should be more reasonable, so that design does not win over accessibility.
